allennlp_models/generation/models/t5.py

```diff
@@ -69,38 +69,38 @@ def tokenizer(self) -> PretrainedTransformerTokenizer:
         return self._tokenizer
 
     def forward(  # type: ignore
-        self, source_tokens: TextFieldTensors, target_tokens: Optional[TextFieldTensors] = None
+        self, source: Dict[str, torch.Tensor], target: Optional[Dict[str, torch.Tensor]] = None
```

Comment:
Are there other uses of this model? Will it upset them?

Reply:
Yea, I should probably create two version of this model to keep backwards compat.